libs/process/background_test.go

```diff
@@ -0,0 +1,31 @@
+package process
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBackground(t *testing.T) {
+	ctx := context.Background()
+	res, err := Background(ctx, []string{"echo", "1"}, WithDir("/"))
+	assert.NoError(t, err)
+	assert.Equal(t, "1", res)
+}
```

Comment:
Separate test where the output goes to stderr.

Reply:
Also a separate test where you run `cat` and expect the output to be empty (because stdin is nil).